### ISSUE TITLE
./wait-for-active should poll /diagstatus on all nodes instead of ./karaf-client 1 showsvcstatus

### ISSUE THREAD

**Issue by vorburger** [jhershberg/netvirt-ha-docker#12]
It would be better if `./wait-for-active` would poll the (new, recent) `/diagstatus` URL on all nodes instead of using `./karaf-client 1 showsvcstatus`. The CLI command on 1 fails if 1 is down and 2 and 3 are up, it's better to poll each node directly, and not rely on 1 to be the orchestrator. There are also "normal" exceptions (shown in [INFRAUTILS-54](https://jira.opendaylight.org/browse/INFRAUTILS-54)) when `showsvcstatus` is used too early during boot.

**Comment by jhershberg** [comment]
Yeah, probably but the output should be parsed because otherwise three responses (or even one) will not fit on a screen.
